mantidproject/mslice
# Propagate font size in generated scripts from plots

**Description of work:**
This PR ensures the title, axis label and colorbar font sizes get propagated when a python script is generated from a plot.

**To test:**
Load data into Mslice
Do a slice plot and cut plot
Change the sizes of the labels
Save the scripts to files
Load these scripts in and run them in Mantid
The font sizes should be propagated

Fixes #795
